src/layman/upgrade/upgrade_v2_0.py

```diff
@@ -0,0 +1,16 @@
+import logging
+
+from db import util as db_util
+from layman import settings
+
+logger = logging.getLogger(__name__)
+DB_SCHEMA = settings.LAYMAN_PRIME_SCHEMA
+
+
+def adjust_db_for_image_mosaic():
+    logger.info(f'    Alter DB prime schema for description')
+
+    statement = f'''
+    ALTER TABLE {DB_SCHEMA}.publications ADD COLUMN IF NOT EXISTS
+    description varchar(1024) default '';'''
```

Comment:
Why is default empty string and not NULL? After previous commit I thought NULL/None is preferred.